Makefile

```diff
@@ -26,11 +26,11 @@ license:
 
 linter: gotools build
 	# TODO enable linter again
-	#@echo "LINT: Running code checks for Go files..."
-	#@cd $$(/bin/pwd) && ./scripts/golinter.sh
+	@echo "LINT: Running code checks for Go files..."
+	@cd $$(/bin/pwd) && ./scripts/golinter.sh
```

Comment:
not sure I like that the implementation links to something in fabric directory. Can we add the linter to our own repo?

Reply:
but then we have to maintain the consistency with the rest of fabric? At least right now before a everything-is-go-modularized, the symlink works rather safely?

Reply:
The main reason why we symlink fabric linters and reuse them is that we dont need to develop our own script nor maintain a copy of the fabric script. This way we can also say that our code passes *all* fabric test and meets the code standard :D